src/components/Dialogs/ConvertTmDialog.jsx

```diff
@@ -106,6 +185,50 @@ const convertTmToTd = (td, htmlInputs) => {
         td = td.split(`{{${key}}}`).join(mappingObject[key])
     })
     const parse = JSON.parse(td);
-    const permalink = `${window.location.href}?td=${encodeURI(JSON.stringify(parse))}`
+
+    // Create new affordances by leaving only the ticked ones
+    if (parse["properties"]) {
+        const newProperties = Object.keys(parse["properties"])
+            .filter(key => properties.includes(key))
+            .reduce((obj, key) => {
+                obj[key] = parse["properties"][key];
+                return obj;
+            }, {});
+
+        // Adapt the new TD
+        parse["properties"] = newProperties;
+    }
+
+    if (parse["actions"]) {
+        const newActions = Object.keys(parse["actions"])
+            .filter(key => actions.includes(key))
+            .reduce((obj, key) => {
+                obj[key] = parse["actions"][key];
+                return obj;
+            }, {});
+
+        // Adapt the new TD
+        parse["actions"] = newActions;
+    }
+
+    if (parse["events"]) {
+        const newEvents = Object.keys(parse["events"])
+            .filter(key => events.includes(key))
+            .reduce((obj, key) => {
+                obj[key] = parse["events"][key];
+                return obj;
+            }, {});
+
+        // Adapt the new TD
+        parse["events"] = newEvents;
+    }
+
+    // Remove TM related data
+    delete parse["@type"];
+    delete parse["tm:required"];
+
+    let permalink = `${window.location.href}?td=${encodeURI(JSON.stringify(parse))}`;
+    // Escaping hash signs manually since they are not escaped automatically
+    permalink = permalink.replace(/#/g, "%23");
```

Comment:
I think this can be done with the function ``encodeURIComponent`` like it is done here. We had the same issue before: 
https://github.com/eclipse/editdor/blob/master/src/components/Dialogs/ShareDialog.jsx#L68

Reply:
Oh, you're right, thank you! I will change it with the next changes to this PR.

Reply:
I've pushed this.